### PR TITLE
feat :[CDS-95242]: Add information regarding using how to use expressions in Ticket Number field in Service Now

### PR DESCRIPTION
**Pull request by krishi0408**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Purpose

What is the motivation for this PR? What changes does it include? Include as much or as little detail as needed.

If this submission relates to a Jira or GitHub issue, include the issue ID in the title.

Internal Harness contributors - Please edit this summary to include preview links after the preview build finishes.

## Merge requirements

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

### CODE REVIEW

**Review by SushrutHarness — DISMISSED** [review]
LGTM

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md [hunk 1]
```diff
@@ -95,6 +95,68 @@ The timezone settings govern the display value of the settings not their actu
 
 6. You can specify additional fields for the ticket by clicking **Provide Field List**. The properties are specified as key-value pairs, the name being the **field name** (not the label) in ServiceNow and a valid value.
 
+## Use an Expressions in Ticket Number
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md:98]:
```suggestion
## Use expressions in Ticket Number
```

Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md:98]: Recommend sentence-style capitalization.
https://harness.atlassian.net/wiki/spaces/DOCS/pages/21544468526/Harness+documentation+style+guide#Headings%2C-titles

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md [hunk 2]
```diff
@@ -95,6 +95,68 @@ The timezone settings govern the display value of the settings not their actu
...
+In Ticket number, you can use an expression or runtime input to reference the Ticket Number from another ``Service Now Create`` step. You can use this Ticket number when you want to update or approve a particular ticket.
+
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md:100]: Recommend using bold for UI elements.
https://harness.atlassian.net/wiki/spaces/DOCS/pages/21544468526/Harness+documentation+style+guide#UI-elements-and-code

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md [hunk 3]
```diff
@@ -95,6 +95,68 @@ The timezone settings govern the display value of the settings not their actu
...
+:::info note
+ The ``Service now Create`` step must be before the ``Service Now Approval`` and ``Service Now Update``.
+:::
+
+Let's say you've set up a [Service Now](/docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-service-now-tickets-in-cd-stages.md) step in your process. Now, you want to use the Harness ``Service Now Approval`` step to give the thumbs-up to the ticket, and once it's approved, you want to use the ``Service Now Update`` step to close it. You'll need to have the ticket number in both the Approval and Update steps to make this happen. In this case, you can make things easier by using the expression type in the Ticket Number field for both the Update and Approval steps.
+
+![](./static/ticket_number.png)
+
+Consider this example yaml:-
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md:109]:
```suggestion
Consider this example YAML:
```

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md [hunk 4]
```diff
@@ -95,6 +95,68 @@ The timezone settings govern the display value of the settings not their actu
...
+                      - name: short_description
+                        value: close the ticket
+                      - name: state
+                        value: "4"
+                  timeout: 10m
+        tags: {}
+
+```
+
+When you run the pipeline and check the input of the ``ServiceNowUpdate_1`` step, you'll notice that the ticket number is retrieved from the expression ``<+execution.steps.create.ticket.ticketNumber>``, as depicted in the below screenshot
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md:157]:
```suggestion
When you run the pipeline and check the input of the ``ServiceNowUpdate_1`` step, you'll notice that the ticket number is retrieved from the expression ``<+execution.steps.create.ticket.ticketNumber>``, as depicted in the below screenshot.
```

**Pull request merged**